lib/sippy_cup/scenario.rb

```diff
@@ -149,13 +149,14 @@ def build(steps)
     # @param [Hash] opts A set of options to modify the message
     # @option opts [Integer] :retrans
     # @option opts [String] :headers Extra headers to place into the INVITE
+    # @option opts [String] :remote_sip_address Remote address to use in place of the SIPp constructed one
```

Comment:
Fairly often with multi-tenant systems, and no, because later expectations/sent messages will revert to the wrong To header.